### ISSUE TITLE
Fraud Detection Text Messages

### ISSUE THREAD

**Issue by Aaeid123** [Aaeid123/Aaeid-Lightwala#3]
`from sklearn.feature_extraction.text import CountVectorizer
from sklearn.model_selection import train_test_split
from sklearn.linear_model import LogisticRegression

text = ["Get free iPhone 13 now!","You won Rs.5000","Your meeting at 8 am","Claim your prize"]
label = [1,1,0,0]

v = CountVectorizer()
X = v.fit_transform(text)
X_train, X_test, y_train, y_test = train_test_split(X,label,test_size=0.2,random_state=42)

model = LogisticRegression()
model.fit(X_train,y_train)
test = ["Your meeting at 9 am"]
test_vector = v.transform(test)
prediction = model.predict(test_vector)

if prediction[0] == 1:
    print("Fraud")

else:
    print("Real")

`
